lib/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
Why do we need this new file lib/index.html?
If you need to change the receiver.html, modify lib/template.html instead.